crates/frontend/src/pages/home.rs

```diff
@@ -66,6 +66,7 @@ pub fn home() -> impl IntoView {
         },
     );
 
+    let (context, set_context) = create_signal::<Vec<(String, String, String)>>(vec![]);
```

Comment:
```suggestion
    let (context_rs, context_ws) = create_signal::<Vec<(String, String, String)>>(vec![]);
```

Reply:
@Datron , leptos follows context and set_context convention in their docs, lets stick with that.

Reply:
@ShubhranshuSanjeev that nomenclature doesn't help identify the type of instrument you are working with. We should self document and differentiate between signals, resources, etc.